testsuite/features/minkvm_guests.feature

```diff
@@ -196,5 +196,6 @@ Feature: Be able to manage KVM virtual machines via the GUI
     And I run "virsh undefine --remove-all-storage test-vm2" on "kvm-server" without error control
     And I run "virsh net-destroy test-net0" on "kvm-server" without error control
     And I run "virsh net-undefine test-net0" on "kvm-server" without error control
+    And I delete all "test-vm.*" volumes from "default" pool on "kvm-server" without error control
```

Comment:
Yes, that's the correct way to do.

Ideally, We would replace all the `I run` steps above with something like `I delete network ...` and `I delete storage ...` where we hide the details of the command we run.

If you have the courage, maybe in another PR...

Reply:
will be in another PR ;)